cluster-scope/base/namespaces/opf-ci-prow/namespace.yaml

```diff
@@ -4,7 +4,17 @@ kind: Namespace
 metadata:
   annotations:
     openshift.io/display-name: "opf-ci-prow"
-    openshift.io/discription: "Project for execution of continous integration prow jobs."
+    openshift.io/discription: "Project for Prow and the gang."
     openshift.io/requester: aicoe-thoth-devops
   name: opf-ci-prow
 spec: {}
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
Please don't define multiple resources in a single file. Please read:

https://github.com/operate-first/apps/blob/master/cluster-scope/README.md

and put this new namespace in `cluster-scope/base/namespaces/opf-ci-test-pods`

You can use the [`hack/onboarding.sh` script](https://github.com/operate-first/apps/blob/master/hack/onboarding.sh)